src/project/mod.rs

```diff
@@ -485,13 +476,22 @@ impl Project {
     }
 }
 
-/// Iterates over the current directory and all its parent directories and returns the first
-/// directory path that contains the [`consts::PROJECT_MANIFEST`].
-pub fn find_project_root() -> Option<PathBuf> {
+/// Iterates over the current directory and all its parent directories and returns the manifest path in the first
+/// directory path that contains the [`consts::PROJECT_MANIFEST`] or [`consts::PYPROJECT_MANIFEST`].
+pub fn find_project_manifest() -> Option<PathBuf> {
     let current_dir = env::current_dir().ok()?;
-    std::iter::successors(Some(current_dir.as_path()), |prev| prev.parent())
-        .find(|dir| dir.join(consts::PROJECT_MANIFEST).is_file())
-        .map(Path::to_path_buf)
+    std::iter::successors(Some(current_dir.as_path()), |prev| prev.parent()).find_map(|dir| {
```

Comment:
We should only except `pyproject.toml` that include the `[tool.pixi]` table. But including the opening in this search might be slow...

Reply:
This is very important for people that do not use pixi to read their pyproject.toml and have them in subdirectories.